src/main/scala/com/linkedin/feathr/offline/config/location/GenericLocation.scala

```diff
@@ -98,30 +98,18 @@ object GenericLocationAdHocPatches {
     location.conf.foreach(e => {
       ss.conf.set(e._1, e._2)
     })
-
-    location.format.toLowerCase() match {
-      case "org.elasticsearch.spark.sql" => {
-        ss.read.format(location.format)
-          .option("es.nodes.wan.only", "true")  // Otherwise Spark will try to find node via broadcast ping, which will always fail
-          .option("pushdown", true) // Enable pushdown to speed up extraction
-          .options(location.options)
-          .load()
-      }
-      case _ => {
-        ss.read.format(location.format)
-          .options(location.options)
-          .load()
-      }
-    }
+    ss.read.format(location.format)
+      .options(location.options)
```

Comment:
Double check with @windoze on this part

Reply:
Checked and updated , pls help review, thx